docker-redbeams/Dockerfile

```diff
@@ -16,7 +17,7 @@ RUN microdnf install unzip
 ADD ${REPO_URL}/com/sequenceiq/redbeams/$VERSION/redbeams-$VERSION.jar /redbeams.jar
 
 # add jmx exporter
-ADD jmx_prometheus_javaagent-0.16.1.jar /jmx_prometheus_javaagent.jar
+RUN curl -o /jmx_prometheus_javaagent.jar https://repo1.maven.org/maven2/io/prometheus/jmx/jmx_prometheus_javaagent/$PROMETHEUS_VERSION/jmx_prometheus_javaagent-$PROMETHEUS_VERSION.jar
```

Comment:
if the agent is installed from maven repo, why is it necessary to have the jar file in the repo? Am I missing something?